lib/Test/MockFile.pm

```diff
@@ -1053,6 +1058,31 @@ sub _goto_is_available {
 }
 
 BEGIN {
+    my $_handle_glob = sub {
+        my $spec = shift;
+
+        # Text::Glob does not understand multiple patterns
+        my @patterns = split /\s+/xms, $spec;
+
+        # Text::Glob does not accept directories in globbing
+        # But csh (and thus, Perl) does, so we need to add them
+        my @mocked_files = keys %files_being_mocked;
```

Comment:
note that a mocked file does not necessary exist

Reply:
maybe:
```perl
my @mocked_files = grep { $files_being_mocked{$_}->exists } keys %files_being_mocked;
```

Reply:
Good catch! Resolved in the branch.